lib/snippets.cson

```diff
@@ -5,13 +5,22 @@
       {
         "${1:.source.js}": {
           "${2:Snippet Name}": {
-            "prefix": "${3:hello}",
+            "prefix": "${3:Snippet Trigger}",
             "body": "${4:Hello World!}"
           }
         }
       }$5
     """
 
+  'Atom Snippet Keys':
```

Comment:
Good one, I think snipns make sense.

About the second one, It's really hard to tell because I think it is very personal. I think that having one "customizable" snippet is better because once you have many snippets it's hard to remember them, so I rather have fewer snippets that can handle different situations than having one for each thing I wanna do. But I do agree that having two of them is more intuitive, specially in this case.